### PR TITLE
scan: implement reward sub and wire up

### LINKED ISSUE #1217: Implement reward sub and wire up

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1217

### CODE REVIEW

File: scan/src/subscriptions/DelegationSub.re [hunk 1]
```diff
@@ -101,12 +111,27 @@ let getTotalStakeByDelegator = delegatorAddress => {
           (),
         ),
     );
-  result
-  |> Sub.map(_, a =>
-       (
-         (a##delegations_view_aggregate##aggregate |> Belt_Option.getExn)##sum |> Belt_Option.getExn
-       )##amount
-     );
+  Js.Console.log(result);
```
Comment (evilpeach) [inline scan/src/subscriptions/DelegationSub.re]: remove log

File: scan/src/subscriptions/DelegationSub.re [hunk 2]
```diff
@@ -88,6 +97,7 @@ let getStakeList = (delegatorAddress, ~page, ~pageSize, ()) => {
           (),
         ),
     );
+  Js.Console.log(result);
```
Comment (evilpeach) [inline scan/src/subscriptions/DelegationSub.re]: remove log too

**Review by evilpeach — CHANGES_REQUESTED** [review]
add reward in `PieChart`

**Review by prin-r — DISMISSED** [review]

File: scan/src/subscriptions/DelegationSub.re [hunk 3]
```diff
@@ -101,12 +110,26 @@ let getTotalStakeByDelegator = delegatorAddress => {
...
+    result
+    |> Sub.map(_, a =>
+         (
+           (a##delegations_view_aggregate##aggregate |> Belt_Option.getExn)##sum
+           |> Belt_Option.getExn
+         )##reward
+       );
+  let%Sub amount = amountSub;
+  let%Sub reward = rewardSub;
+
```
Comment (evilpeach) [inline scan/src/subscriptions/DelegationSub.re]: Why don't use a single subscription? 
Like this
```
let delegatorInfoSub = result
    |> Sub.map(_, a =>
         (
           (a##delegations_view_aggregate##aggregate |> Belt_Option.getExn)##sum
           |> Belt_Option.getExn
         )

let%Sub delegatorInfo = delegatorInfoSub;
// And then return
{amount: delegatorInfo##amount, reward: delegatorInfo##reward} |> Sub.resolve;
```

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**